docs/workloads/azure/frontend/index.md

```diff
@@ -24,7 +24,7 @@ The base domain code explanations and usage guides for extending can be found
 ---
 
 The base deployment code explanations and usage guides for extending can be
-found [here](./deployment.md)
+found [here](deployment.md)
 
 ## Infrastructure Code
```

Comment:
Is this file used anywhere?

Reply:
I haven't seen the index page referenced anywhere.

Reply:
I believe it can be deleted. It might have been there since before we started implementing the changes as part of cycle 5.